envoy/server/instance.h

```diff
@@ -269,6 +275,12 @@ class Instance {
    * TODO(mattklein123): This can be removed when version 1.20.0 is no longer supported.
    */
   virtual bool enableReusePortDefault() PURE;
+
+  /**
+   * Set predicates for filtering stats to be flushed to sinks.
```

Comment:
Does this apply to histograms? My guess is no (because they must be sinked to work properly, I believe). If that's the case, please document that.

Reply:
Modified comment to indicate this works for counters, gauges and text readouts. 
I will add the ability to set predicates for histograms in a follow up PR. That will only affect which histograms are available in the MetricSnapshot. The histogram merge behavior will be unchanged.